Projects/Login/LoginData/Sources/DTO/Request/AppleLoginRequestDTO.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  AppleLoginRequestDTO.swift
+//  Repository
+//
+//  Created by SiJongKim on 6/30/25.
+//  Copyright © 2025 com.noweekend. All rights reserved.
+//
+
+import Foundation
+
+public struct AppleLoginRequestDTO: Encodable {
+    public let authorizationCode: String
+    public let name: String?
```

Comment:
```suggestion
    let authorizationCode: String
    let name: String?
```
생성자로 받아서 생성하고, 바로data타입으로 사용할거같아서
외부에서 읽을거아니면 public할 필요는 없을거같아여!